tsdfileapi/test_file_api.py

```diff
@@ -1860,12 +1884,121 @@ def test_nacl_crypto(self):
         )
         assert decrypted_payload.decode() == payload
 
-        # now with requests
-        # get server public key
-        # send data
-        # - files
-        # - tables enc({}), enc([{}, {}]), [end({})] ?
-        # - figure out content-types
+        # write an encryptde stream to a file (save a stream)
+        chunk_size = 10
+        smaller = []
+        larger = []
+        with open('/tmp/lines', 'w') as f:
```

Comment:
Consider using `tempfile.NamedTemporaryFile` instead of hardcoding a temporary file path.

Reply:
will check it out

Reply:
will address it in a future PR, moving on to finish the json part of this feature for the moment